Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.cs

```diff
@@ -99,144 +99,144 @@ public void SetConverterSettings(object settings)
-        case DB.Area o:
-          returnObject = AreaToSpeckle(o);
-          break;
-        case DB.Architecture.Room o:
-          returnObject = RoomToSpeckle(o);
-          break;
-        case DB.Architecture.TopographySurface o:
-          returnObject = TopographyToSpeckle(o);
-          break;
-        case DB.Wall o:
-          returnObject = WallToSpeckle(o);
-          break;
-        case DB.Mechanical.Duct o:
-          returnObject = DuctToSpeckle(o);
-          break;
-        case DB.Mechanical.FlexDuct o:
-          returnObject = DuctToSpeckle(o);
-          Report.Log($"Converted FlexDuct {o.Id}");
-          break;
-        case DB.Mechanical.Space o:
-          returnObject = SpaceToSpeckle(o);
-          break;
-        case DB.Plumbing.Pipe o:
-          returnObject = PipeToSpeckle(o);
-          break;
-        case DB.Plumbing.FlexPipe o:
-          returnObject = PipeToSpeckle(o);
-          Report.Log($"Converted FlexPipe {o.Id}");
-          break;
-        case DB.Electrical.Wire o:
-          returnObject = WireToSpeckle(o);
-          break;
-        case DB.Electrical.CableTray o:
-          returnObject = CableTrayToSpeckle(o);
-          break;
-        //these should be handled by curtain walls
-        case DB.CurtainGridLine _:
-          returnObject = null;
-          break;
-        case DB.Architecture.BuildingPad o:
-          returnObject = BuildingPadToSpeckle(o);
-          break;
-        case DB.Architecture.Stairs o:
-          returnObject = StairToSpeckle(o);
-          break;
-        //these are handled by Stairs
-        case DB.Architecture.StairsRun _:
-          returnObject = null;
-          break;
-        case DB.Architecture.StairsLanding _:
-          returnObject = null;
-          break;
-        case DB.Architecture.Railing o:
-          returnObject = RailingToSpeckle(o);
-          break;
-        case DB.Architecture.TopRail _:
-          returnObject = null;
-          break;
-        case DB.Structure.Rebar o:
-          returnObject = RebarToSpeckle(o);
-          break;
-        case DB.Ceiling o:
-          returnObject = CeilingToSpeckle(o);
-          break;
-        case DB.PointCloudInstance o:
-          returnObject = PointcloudToSpeckle(o);
-          break;
-        case DB.ProjectInfo o:
-          returnObject = ProjectInfoToSpeckle(o);
-          break;
-        case DB.ElementType o:
-          returnObject = ElementTypeToSpeckle(o);
-          break;
-        case DB.Grid o:
-          returnObject = GridLineToSpeckle(o);
-          break;
-        case DB.ReferencePoint o:
-          if ((BuiltInCategory)o.Category.Id.IntegerValue == BuiltInCategory.OST_AnalyticalNodes)
-          {
-            returnObject = AnalyticalNodeToSpeckle(o);
-
-          }
-          break;
-        case DB.Structure.BoundaryConditions o:
-          returnObject = BoundaryConditionsToSpeckle(o);
-          break;
+        public Base ConvertToSpeckle(object @object)
```

Comment:
Seems like your IDE is using 4 space indent while we're using 2 space indent. Could you unindent these?
This should be being picked up by your IDE from our .editorconfig file but it appears they're not.

Not a big deal though! Just trying to keep things clean :)

Reply:
Hey @AlanRynne

Thanks for reviewing. I tried to find the ```.editorconfig```. But it seems to be missing in ```Òbjects```

I changed the indent from 4 to 2 and autoformatted again. It looks like a lot has changed. But it didnn't and the formating is fixed now.

Reply:
Great! That looks much lighter now 👍🏻 I'll let @connorivy give you the go-ahead on the Revit changes :)